lib/packwerk/cli.rb

```diff
@@ -117,6 +120,11 @@ def generate_configs
     end
 
     def update(paths)
+      warn("`packwerk update` will be deprecated in favor of `packwerk update-deprecations`.")
```

Comment:
```suggestion
      warn("`packwerk update` is deprecated in favor of `packwerk update-deprecations`.")
```

AFAIK `deprecated` is already forward-looking; it means that it's going to be removed in the future